### PR TITLE
Tech tasks

### PR DESCRIPTION
**Pull request by giladshanan**
# Release Notes

A collection of small clean up tasks that have been sitting in my git stash

### CODE REVIEW

File: app/views/statements/list.html.haml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- # This template appears to be unused.  If nothing is logged to rollbar by June 2023, delete this file.
+- # TO DO: This template appears to be unused.  If nothing is logged to rollbar by June 2023, delete this file.
```
Comment (LeticiaErrandonea) [inline app/views/statements/list.html.haml:1]: Why adding "TO DO" instead of deleting the file?

File: app/models/price_policy.rb [hunk 2]
```diff
@@ -203,6 +192,17 @@ def start_date_is_unique
     end
   end
 
+  def expire_date_within_fiscal_year
+    if expire_date.present? && start_date.present?
+      gen_exp_date = generate_expire_date(start_date)
```
Comment (LeticiaErrandonea) [inline app/models/price_policy.rb:197]: Tests are failing because `generate_expire_date` is a class method

**Review by LeticiaErrandonea — COMMENTED** [review]
Left a couple of comments.

Also, there are some conflicts

File: app/views/statements/list.html.haml (hunk 1 above)
Comment (giladshanan) [reply app/views/statements/list.html.haml:1]: Good question :) I checked rollbar and confirmed there are no warnings here, so I'll go ahead and remove it 🗑️

**Review by LeticiaErrandonea — APPROVED** [review]
LGTM, thanks for the changes!

**Comment by giladshanan** [comment]
@LeticiaErrandonea thanks for the review!

**Pull request merged**